recipes/opensartoolkit/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "opensartoolkit" %}
+{% set version = "0.12.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/opensartoolkit-{{ version }}.tar.gz
+  sha256: 867feb004333beb8245cbb6c548753bdc4d35545b99ee4496013a830442ea145
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.6
+    - setuptools
+    - setuptools
+  run:
+    - python >=3.6
+    - descartes
+    - fiona
+    - gdal >=2
+    - geopandas >=0.8
+    - godale
+    - imageio
+    - jupyterlab
+    - matplotlib-base
+    - numpy
+    - pandas
+    - psycopg2-binary
+    - pyproj >=2.1
+    - python
+    - rasterio
+    - requests
+    - retrying
+    - rtree
+    - scipy
+    - shapely
+    - tqdm
+
+test:
+  imports:
+    - ost
+    - ost.generic
+  commands:
+    - pip check
```

```suggestion
  imports:
    - ost.generic
  commands:
    - pip check
    - GPT_PATH=" " python -c "import ost"
```

```suggestion
```

Can you delete `import ost` here? If you try it locally, you get a prompt on linux:

```
>>> import ost
 Please provide the full path to the SNAP gpt command line executable (e.g. /path/to/snap/bin/gpt) or leave empty if you just want to use the OST inventory and download routines.
```

Instead, you could just declare an env variable like this: `GPT_PATH=" " python -c "import ost"` under commands.

---

This will fix the tests for linux, I believe